packages/cli/src/commands/test.js

```diff
@@ -134,6 +137,49 @@ export const handler = async ({
     jestArgs.push('--projects', ...sides)
   }
 
+  try {
+    if (sides.includes('api')) {
+      if (!fs.existsSync(path.join(rwjsPaths.api.base, 'jest.config.js'))) {
+        console.error(
+          c.error('Error: Jest config file not found in `api/jest.config.js.`')
+        )
+        console.error(
+          c.error(
+            'Run codemod `npx @redwoodjs/codemods@latest update-jest-config`.'
+          )
+        )
+        console.error(
+          c.error(
+            'This command will automatically update your project to the latest Jest config.'
+          )
+        )
+        throw new Error('Error: Jest config file not found in api side')
```

Comment:
I think this is a good opportunity for DRY programming — create a function and pass in the side. Also, you might just have one `console.error` and use line breaks `\n` (template literal should add line breaks if you separate onto lines, I believe).

I would definitely add line break before and after the console error. Visually it helps to make the output separate. And you can consolidate into two lines with something like this:
```
Error: Jest config file not found `api/jest.config.js`
To add this file, run `npx @redwoodjs/codemods update-jest-config`
```